tests/ui/loop-match/const-continue-outside-loop-match.rs

```diff
@@ -0,0 +1,62 @@
+// Test that the correct error is emitted when `#[const_continue]` occurs outside of
+// a loop match. See also https://github.com/rust-lang/rust/issues/143165.
+#![allow(incomplete_features)]
+#![feature(loop_match)]
+#![crate_type = "lib"]
+
+fn main1() {
+    let mut state;
+    loop {
+        state = 'state: {
+            #[const_continue]
+            break break;
```

Comment:
```suggestion
        state = 'state: {
            #[const_continue]
            break 'state ();
```

"unlabeled `break`" errors are irrelevant, the never type is probably also irrelevant.